src/main/java/thePackmaster/packs/DefectPack.java

```diff
@@ -26,9 +26,9 @@ public ArrayList<String> getCards() {
         cards.add(Hologram.ID);
         cards.add(Defragment.ID);
         cards.add(Sunder.ID);
-        cards.add(Chaos.ID);
+        cards.add(GoForTheEyes.ID);
         cards.add(Fusion.ID);
-        cards.add(Skim.ID);
```

Comment:
Noooooooooooooooooooooooooooooooooooooooooooooooooo my card draw.

It's fine, just that I like Skim. Most other packs have enough card draw in them anyway.